### DISCUSSION TITLE
Unable to convert R object to python type - callpeak

### DISCUSSION THREAD

**Discussion by JonathanNoonan** [stuart-lab/signac#736] [General]
Hi All,

I'm getting the following error when I try to run the callpeak function:

Error in py_call_impl(callable, dots$args, dots$keywords) : 
  Evaluation error: Unable to convert R object to Python type.

I have no experience using python so I'm not sure what the issue is. Help would be much appreciated!

Thanks

**Comment by timoast** [comment]
Please open an issue and include the full code you're running and the output of `sessionInfo()`
